src/connect-zbt-1/getting-started/migrate-existing-zha-network-from-backup.md

```diff
@@ -18,74 +18,129 @@ Please note that not all settings can be migrated in this case:
 
 ## To migrate an existing Zigbee Home Automation (ZHA) network via Zigbee2MQTT backup
 
-1. Creating a Zigbee2MQTT backup
-   1. Under **Settings** > **Add-ons**, open the **Zigbee2MQTT** add-on and select **Open Web UI**.
-   2. In the Zigbee2MQTT UI, go to **Settings** > **Tools** and select **Request z2m backup**.
-      - A zip file should now be downloaded into your **Downloads** folder.
+{% steps %}
+{% step "Creating a Zigbee2MQTT backup" %}
+{% image "/static/img/connect-zbt-1/z2m-backup-01.png" "Creating a Zigbee2MQTT backup" %}
+{% stepContent %}
 
-   ![/static/img/connect-zbt-1/z2m-backup-01.png](/static/img/connect-zbt-1/z2m-backup-01.png)
+1. Under **Settings** > **Add-ons**, open the **Zigbee2MQTT** add-on and select **Open Web UI**.
+2. In the Zigbee2MQTT UI, go to **Settings** > **Tools** and select **Request z2m backup**.
+   - {% stepResult "A zip file should now be downloaded into your **Downloads** folder." %}
 
-2. Unzipping the backup folder.
-   - In your **Downloads** folder, unzip the **z2m-backup**.
-     - It should contain a **coordinator_backup.json** file.
+{% endstepContent %}
+{% endstep %}
 
-   ![/static/img/connect-zbt-1/z2m-backup-restore-01.png](/static/img/connect-zbt-1/z2m-backup-restore-01.png)
+{% step "Unzipping the backup folder" %}
+{% image "/static/img/connect-zbt-1/z2m-backup-restore-01.png" "Unzipping the backup folder" %}
+{% stepContent %}
 
-3. Stopping the Zigbee2MQTT add-on.
-   - Under **Settings** > **Add-ons**, in the **Zigbee2MQTT** add-on, select **Stop**.
-   - Zigbee2MQTT uses the MQTT discovery feature to make the Zigbee devices available in Home Assistant. After removing the Zigbee2MQTT add-on, the devices are still present in Home Assistant since the MQTT broker is still holding on to the MQTT discovery messages.
-   - It is recommended to remove these stale devices from the MQTT broker (see next steps).
-   ![/static/img/connect-zbt-1/z2m-delete-add-on.png](/static/img/connect-zbt-1/z2m-delete-add-on.png)
+- In your **Downloads** folder, unzip the **z2m-backup**.
+  - It should contain a **coordinator_backup.json** file.
```

Comment:
```suggestion
{% stepInfo "Notice the **coordinator_backup.json** file. %}
```

Reply:
thank you for the catch :fishing_pole_and_fish:  https://github.com/NabuCasa/support/pull/497